docs/guide/testing.md

```diff
@@ -128,13 +128,31 @@ To use RSpec, add the following:
 
 ```ruby
 require "view_component/test_helpers"
+require "capybara/rspec"
 
 RSpec.configure do |config|
   config.include ViewComponent::TestHelpers, type: :component
+  config.include Capybara::RSpecMatchers, type: :component
 end
 ```
 
-Specs created by the generator have access to test helpers like `render_inline`.
+Specs created by the generator have access to test helpers like `render_inline`. For example:
+
+```ruby
+require "rails_helper"
+
+RSpec.describe ExampleComponent, type: :component do
+  subject { rendered_component }
+
+  it "renders component" do
+    render_inline(described_class.new(title: "my title")) { "Hello, World!" }
+
+    is_expected.to have_css "span[title='my title']", text: "Hello, World!"
+    # or, to just assert against the text
+    is_expected.to have_text "Hello, World!"
```

Comment:
```suggestion
    expect(rendered_component).to have_css "span[title='my title']", text: "Hello, World!"
    # or, to just assert against the text
    expect(rendered_component).to have_text "Hello, World!"
```

I think it may be valuable to avoid using `subject` and `is_expected` here, so the test can be a bit more explicit about what we're asserting against.